integration-tests/suites/runtime_config_file.go

```diff
@@ -0,0 +1,172 @@
+package suites
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/stackrox/collector/integration-tests/pkg/assert"
+	"github.com/stackrox/collector/integration-tests/pkg/collector"
+	"github.com/stackrox/collector/integration-tests/pkg/common"
+	"github.com/stackrox/collector/integration-tests/pkg/config"
+	"github.com/stackrox/collector/integration-tests/pkg/types"
+)
+
+type RuntimeConfigFileTestSuite struct {
+	IntegrationTestSuiteBase
+}
+
+func (s *RuntimeConfigFileTestSuite) createDirectory(dir string) {
+	cmd := "mkdir " + dir
+	s.execShellCommand(cmd)
+}
+
+func (s *RuntimeConfigFileTestSuite) deleteFile(file string) {
+	cmd := "rm " + file
+	s.execShellCommand(cmd)
+}
+
+func (s *RuntimeConfigFileTestSuite) setRuntimeConfig(runtimeConfigFile string, configStr string) {
+	cmd := "echo '" + configStr + "' > " + runtimeConfigFile
+	s.execShellCommand(cmd)
+}
+
+func (s *RuntimeConfigFileTestSuite) setExternalIpsEnable(runtimeConfigFile string, enable bool) {
+	configStr := "networking:\n  externalIps:\n    enable: " + strconv.FormatBool(enable)
```

Comment:
```suggestion
	configStr := `networking:
  externalIps:
    enable: ` + strconv.FormatBool(enable)
```

Or even better, use the `RuntimeConfig` type defined in `integration-tests/pkg/types/runtime_config.go` and serialize it to yaml.